src/funfuzz/js/js_interesting.py

```diff
@@ -71,7 +70,7 @@ def __init__(self, options, runthis, logPrefix, in_compare_jit):  # pylint: disa
         if options.valgrind:
             runthis = (
                 inspect_shell.constructVgCmdList(errorCode=VALGRIND_ERROR_EXIT_CODE) +
```

Comment:
Minor nit but string concatenation is ugly.  Why not store them in a list and return ''.join(list)?

Reply:
This is list concatenation, not string concat. `constructVgCmdList` and `valgrindSuppressions()` both return lists.